### DISCUSSION TITLE
Spock 2.4-M1

### DISCUSSION THREAD

**Discussion by leonard84** [spockframework/spock#1555] [Releases]
https://spockframework.org/spock/docs/2.4-M1/release_notes.html

<hr /><em>This discussion was created from the release <a href='https://github.com/spockframework/spock/releases/tag/spock-2.4-M1'>Spock 2.4-M1</a>.</em>

**Comment by radarsh** [comment]
@leonard84 When can we have a GA release for this version? Or another cut in the 2.3 line? Many of us are interested in several bugs that were fixed in 2.4-M1 that haven't yet been released, specifically https://github.com/spockframework/spock/issues/1539 especially now that Spring Boot 3 is gaining popularity across organisations.
